martian-derive/tests/mro/test_empty_split.mro

```diff
@@ -6,10 +6,10 @@
 #
 
 stage CHUNK_READS(
-    in  map[] chunks,
-    in  int   reads_per_file,
-    out map[] out_chunks,
-    src comp  "my_adapter martian chunker",
+    in  map<int>[] chunks,
+    in  int        reads_per_file,
+    out map<int>[] out_chunks,
+    src comp       "my_adapter martian chunker",
```

Comment:
Side note: perhaps it would be a good idea to include a test that the output mro is unchanged by `mro format`.  This could be a separate CI test action, just downloading the [martian release tarball](https://github.com/martian-lang/martian/releases/download/v4.0.3/martian-v4.0.3-linux-x86_64.tar.xz) to get the tool and verifying these files.